app/templates/dashboard_single_fund.html

```diff
@@ -31,7 +31,7 @@ <h2 class="govuk-heading-l">
                 {% endif %}
                     {{ applications_table(round["applications"], round["is_past_submission_deadline"], show_language_column) }}
                 {% if not round["is_past_submission_deadline"] %}
-                    {{ startApplicationButton(form, url_for('account_routes.new'), fund_id=fund["fund_id"], round_id=round["round_details"]["id"], existing_applications_count=round["applications"]|count) }}
+                    {{ startApplicationButton(form, url_for('account_routes.new'), fund_id=fund["fund_data"]["id"], round_id=round["round_details"]["id"], existing_applications_count=round["applications"]|count) }}
                 {% endif %}
             {% endif %}
         {% endfor %}
```

Comment:
I had the check this with Alan but...
_In general, it is not recommended to include complex logic or access complex data structures such as dictionaries or arrays directly within templates in most templating engines. This is because templates are meant to be focused on rendering and presentation logic, while the business logic and data manipulation should be handled outside of the template.

Instead, it is better to provide the template with the variables or data it needs from the external context, usually through a separate controller or view function. In this case, it would be preferable to provide the template with the variable fund_id directly from the controller or view function, rather than accessing fund["fund_data"]["id"] directly within the template._

Reply:
Ah! although I do agree with Alan, this is the simplest (and i think only) way for us to loop through each round and use the appropriate/ dynamic IDs for the urls. We could generate all our info needed in the function calling the template but this will still have to be stored in a complex data structure for us to dynamically call the right IDs... 😢